# 16S-rRNA_SITES

## TO KNOW ABOUT 16S rRNA:

[Structure of 30S ribosomal unit](https://www.esrf.fr/UsersAndScience/Publications/Highlights/2000/life-sci/LS6.html)

[A complex assembly landscape for the 30S ribosomal subunit](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5654522/)

[small ribosomal unit with good image](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.0020010)

[16S rRNA bioinformatics analysis pipeline](https://www.cd-genomics.com/16S-18S-ITS-Amplicon-Sequencing.html)

[A little explanation of the functions from the given material](https://en.wikipedia.org/wiki/16S_ribosomal_RNA)

[The distribution, diversity, and importance of 16S rRNA gene introns in the order Thermoproteales](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4496867/)

## ANALYSIS SITES

[Analysing Microbial Community Composition through Amplicon Sequencing: From Sampling to Hypothesis Testing](https://www.frontiersin.org/articles/10.3389/fmicb.2017.01561/full)

[Microbial Diversity Analysis â€“ 16S/18S/ITS Sequencing](https://www.cd-genomics.com/microbioseq/microbial-diversity-analysis.html)

[A practical guide to amplicon and metagenomic analysis of microbiome data](https://link.springer.com/article/10.1007/s13238-020-00724-8#Sec3)

[Metagenomic 16S rDNA amplicon data of microbial diversity and its predicted metabolic functions in the Southern Ocean (Antarctic)](https://www.researchgate.net/publication/337477156_Metagenomic_16S_rDNA_amplicon_data_of_microbial_diversity_and_its_predicted_metabolic_functions_in_the_Southern_Ocean_Antarctic)

[Metabolic pathways inferred from a bacterial marker gene illuminate ecological changes across South Pacific frontal boundaries](https://www.nature.com/articles/s41467-021-22409-4?proof=t%C2%A0#Sec23)
* [SqueezeMeta: a fully automated metagenomics pipeline, from reads to bins](https://github.com/jtamames/SqueezeMeta#squeezemeta-a-fully-automated-metagenomics-pipeline-from-reads-to-bins)

[16S rRNA gene amplicon-based metagenomic analysis of bacterial communities in the rhizospheres of selected mangrove species from Mida Creek and Gazi Bay, Kenya](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0248485)

[A practical guide to ampliconand metagenomic analysis of microbiome data](https://www.researchgate.net/publication/341307321_A_practical_guide_to_amplicon_and_metagenomic_analysis_of_microbiome_data)

[Comparative analysis of amplicon and metagenomic sequencing methods reveals key features in the evolution of animal metaorganisms](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-019-0743-1#Sec13)

