# 16S-rRNA_SITES

## TO KNOW ABOUT 16S rRNA:

[Structure of 30S ribosomal unit](https://www.esrf.fr/UsersAndScience/Publications/Highlights/2000/life-sci/LS6.html)

[A complex assembly landscape for the 30S ribosomal subunit](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5654522/)

[small ribosomal unit with good image](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.0020010)

[16S rRNA bioinformatics analysis pipeline](https://www.cd-genomics.com/16S-18S-ITS-Amplicon-Sequencing.html)

[A little explanation of the functions from the given material](https://en.wikipedia.org/wiki/16S_ribosomal_RNA)

[The distribution, diversity, and importance of 16S rRNA gene introns in the order Thermoproteales](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4496867/)

## ANALYSIS SITES

[Analysing Microbial Community Composition through Amplicon Sequencing: From Sampling to Hypothesis Testing](https://www.frontiersin.org/articles/10.3389/fmicb.2017.01561/full)

[Microbial Diversity Analysis â€“ 16S/18S/ITS Sequencing](https://www.cd-genomics.com/microbioseq/microbial-diversity-analysis.html)

[A practical guide to amplicon and metagenomic analysis of microbiome data](https://link.springer.com/article/10.1007/s13238-020-00724-8#Sec3)

[Metagenomic 16S rDNA amplicon data of microbial diversity and its predicted metabolic functions in the Southern Ocean (Antarctic)](https://www.researchgate.net/publication/337477156_Metagenomic_16S_rDNA_amplicon_data_of_microbial_diversity_and_its_predicted_metabolic_functions_in_the_Southern_Ocean_Antarctic)

[Metabolic pathways inferred from a bacterial marker gene illuminate ecological changes across South Pacific frontal boundaries](https://www.nature.com/articles/s41467-021-22409-4?proof=t%C2%A0#Sec23)
* [SqueezeMeta: a fully automated metagenomics pipeline, from reads to bins](https://github.com/jtamames/SqueezeMeta#squeezemeta-a-fully-automated-metagenomics-pipeline-from-reads-to-bins)

[16S rRNA gene amplicon-based metagenomic analysis of bacterial communities in the rhizospheres of selected mangrove species from Mida Creek and Gazi Bay, Kenya](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0248485)

[Comparative analysis of amplicon and metagenomic sequencing methods reveals key features in the evolution of animal metaorganisms](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-019-0743-1#Sec13)

[A perspective on 16S rRNA operational taxonomic unit clustering using sequence similarity](https://www.nature.com/articles/npjbiofilms20164)

[Interpretations of Environmental Microbial Community Studies Are Biased by the Selected 16S rRNA (Gene) Amplicon Sequencing Pipeline](https://www.frontiersin.org/articles/10.3389/fmicb.2020.550420/full)

[Reprocessing 16S rRNA Gene Amplicon Sequencing Studies: (Meta)Data Issues, Robustness, and Reproducibility](https://www.frontiersin.org/articles/10.3389/fcimb.2021.720637/full)

[Impact of DNA extraction method and targeted 16S-rRNA hypervariable region on oral microbiota profiling](https://www.nature.com/articles/s41598-018-34294-x)

[Amplicon Sequencing of Variable 16S rRNA from Bacteria andITS2 Regions from Fungi and Plants, Reveals HoneybeeSusceptibility to Diseases Results from Their ForageAvailability under Anthropogenic Landscapes](https://www.mdpi.com/2076-0817/10/3/381/pdf)

[Considerations For Optimizing Microbiome Analysis Using a Marker Gene](https://www.frontiersin.org/articles/10.3389/fnut.2016.00026/full)

[16S rRNA Amplicon Sequencing for Epidemiological Surveys of Bacteria in Wildlife](https://journals.asm.org/doi/10.1128/mSystems.00032-16)


## THE OTU

[Operational Taxonomic Unit](https://www.sciencedirect.com/topics/medicine-and-dentistry/operational-taxonomic-unit)

[Operational taxonomic unit (OTU)](https://www.metagenomics.wiki/pdf/definition/operational-taxonomic-unit-otu)

[Current challenges and best-practice protocols for microbiome analysis](https://academic.oup.com/bib/article/22/1/178/5678919)

[A comprehensive method for amplicon-based and metagenomic characterization of viruses, bacteria, and eukaryotes in freshwater samples](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-016-0166-1)

[Considerations For Optimizing Microbiome Analysis Using a Marker Gene](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4976105/)

[Controlling for Contaminants in Low-Biomass 16S rRNA Gene Sequencing Experiments](https://journals.asm.org/doi/10.1128/mSystems.00290-19)

[iMAP: an integrated bioinformatics and visualization pipeline for microbiome data analysis](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-019-2965-4)

[tidyMicro: a pipeline for microbiome data analysis and visualization using the tidyverse in R](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-021-03967-2#Sec2)

[BiomMiner: An advanced exploratory microbiome analysis and visualization pipeline](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0234860)

[Analysis of the mouse gut microbiome using full-length 16S rRNA amplicon sequencing](https://www.researchgate.net/publication/305336008_Analysis_of_the_mouse_gut_microbiome_using_full-length_16S_rRNA_amplicon_sequencing)

[Template Preparation Affects 16S rRNA High-Throughput Sequencing Analysis of Phyllosphere Microbial Communities](https://www.frontiersin.org/articles/10.3389/fpls.2017.01623/full)

[rANOMALY: AmplicoN wOrkflow for Microbial community AnaLYsis](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7836088/)

[Analysis of large 16S rRNA Illumina data sets: Impact of singleton read filtering on microbial community description](https://onlinelibrary.wiley.com/doi/10.1111/1755-0998.12700)

